modules/core/src/main/java/org/apache/ignite/internal/suggestions/OsConfigurationSuggestions.java

```diff
@@ -60,12 +60,12 @@
     public static synchronized List<String> getSuggestions() {
         List<String> suggestions = new ArrayList<>();
 
-        if (U.isRedHat()) {
-            String value;
-            String expected = "500";
+        if (U.isLinux()) {
+            String val;
+            String exp = "500";
 
-            boolean dwcParamFlag = (value = readVmParam(DIRTY_WRITEBACK_CENTISECS)) != null && !value.equals(expected);
-            boolean decParamFlag = (value = readVmParam(DIRTY_EXPIRE_CENTISECS)) != null && !value.equals(expected);
+            boolean dwcParamFlag = (val = readVmParam(DIRTY_WRITEBACK_CENTISECS)) != null && !val.equals(exp);
```

Comment:
Can we parse the value as int and do a "less than" comparison? val < exp (500)
Otherwise we may be telling people to slow down their writeback. Applies for both.

Reply:
Cast exp and val to int and replaced .equals comparison with val > exp (500)
The gridgain guide suggests: As a solution, you can try changing the page flushing interval from the default 30 seconds to 5 seconds